test/modellib/RNNModel.cpp

```diff
@@ -46,13 +46,13 @@ bool RNNLibBuilder::build() {
   if (isDynamicB)
     B1 = -1;
 
-  xShape = {S, B, I};
-  llvm::SmallVector<int64_t, 3> xShapeSymbol = {S1, B1, I};
-  llvm::SmallVector<int64_t, 3> wShape = {D, H, I};
-  llvm::SmallVector<int64_t, 3> rShape = {D, H, H};
-  llvm::SmallVector<int64_t, 2> bShape = {D, 2 * H};
-  hShape = {D, B, H};
-  llvm::SmallVector<int64_t, 3> hShapeSymbol = {D, B1, H};
+  xShape = perm3(S, B, I);
+  std::vector<int64_t> xShapeSymbol = perm3(S1, B1, I);
```

Comment:
We try to avoid `std::vector` as much as possible for a small container and use `SmallVector` for better performance. Could you keep using `SmallVector` here? For example, using something like this instead:
```C
llvm::SmallVector<int64_t, 3> xShapeSymbol = {S1, B1, I};
perm3(xShapeSymbol)
```

Reply:
@tungld ok, I changed back to SmallVector, PTAL

Reply:
Thank you very much!